### PR TITLE
⏺️🎵 🎵  ⏺️ Load More records ⏺️🎵🎵  ⏺️ 

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- Add button to load more contacts 🎵 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/src/components/contacts/Contacts.js [hunk 1]
```diff
@@ -9,11 +9,16 @@ import Customers from '../customers/Customers';
...
+const getContacts = (setContacts, setAlertMessage, setNextCursor) => {
   axios
     .get("/contacts")
     .then((res) => {
       setContacts(res.data.items);
+      if(res.data.metadata) {
+        setNextCursor(res.data.metadata.nextCursor);
+      } else {
+        setNextCursor("")
+      }
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:21]:
```suggestion
    .then(({ data: { items, metadata } }) => {
      setContacts(items);
      if (metadata) {
        setNextCursor(metadata.nextCursor);
      } else {
        setNextCursor("");
      }
```

File: src/frontend/src/components/contacts/Contacts.js [hunk 2]
```diff
@@ -35,8 +40,29 @@ const deleteContact = (contactToDelete, setShowModal, setAlertMessage, setContac
     })
 };
 
+const loadMoreContact = (nextCursor, setContacts, setNextCursor, setAlertMessage) => {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:43]:
```suggestion
const loadMoreContacts = (nextCursor, setContacts, setNextCursor, setAlertMessage) => {
```

File: src/frontend/src/components/contacts/Contacts.js [hunk 3]
```diff
@@ -35,8 +40,29 @@ const deleteContact = (contactToDelete, setShowModal, setAlertMessage, setContac
...
+
+  if(nextCursor !== "") {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:45]:
```suggestion
  if (nextCursor !== "") {
```

File: src/frontend/src/components/contacts/Contacts.js [hunk 4]
```diff
@@ -35,8 +40,29 @@ const deleteContact = (contactToDelete, setShowModal, setAlertMessage, setContac
...
+    axios
+      .get("/contacts?fromcursor=" + nextCursor)
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:47]:
```suggestion
      .get(`/contacts?fromcursor=${nextCursor}`)
```

File: src/frontend/src/components/contacts/Contacts.js [hunk 5]
```diff
@@ -35,8 +40,29 @@ const deleteContact = (contactToDelete, setShowModal, setAlertMessage, setContac
...
+      .then((res) => {
+        setContacts(contacts => contacts.concat(res.data.items));
+        if(res.data.metadata) {
+          setNextCursor(res.data.metadata.nextCursor)
+        } else {
+          setNextCursor("")
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:53]:
```suggestion
      .then(({ data: { items, metadata } }) => {
        setContacts(contacts => contacts.concat(items));
        if (metadata) {
          setNextCursor(metadata.nextCursor);
        } else {
          setNextCursor("");
```

File: src/frontend/src/components/contacts/Contacts.js [hunk 6]
```diff
@@ -135,6 +161,10 @@ export default function Contacts() {
         </>
       )}
       <br />
+      { nextCursor !== "" && (
+        <Button onClick={() => loadMoreContact(nextCursor, setContacts, setNextCursor, setAlertMessage)} label="Load more contacts" styling="normal-blue btn" />
+      )}
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/src/components/contacts/Contacts.js:166]:
```suggestion
      {nextCursor !== "" && (
        <Button onClick={() => loadMoreContacts(nextCursor, setContacts, setNextCursor, setAlertMessage)} label="Load more contacts" styling="normal-blue btn" />
      )}
```

**Review by sdevalapurkar-bcgov — CHANGES_REQUESTED** [review]
awesome work and cool changes!

File: src/frontend/src/components/contacts/Contacts.js (hunk 4 above)
Comment (alexjoybc) [reply src/frontend/src/components/contacts/Contacts.js:47]: way better lol

**Review by sdevalapurkar-bcgov — APPROVED** [review]
LGTM!

**Comment by sdevalapurkar-bcgov** [comment]
nice PR title!

**Pull request merged**